test/__init__.py

```diff
@@ -1,15 +1 @@
-import os
-import subprocess
-import sys
 
-import libfaketime
-
-
-def setup_package():
```

Comment:
I think I added the recompile here to make things easier while working on the c code.

Thoughts on making it opt-in (eg an env var, or maybe pytest has something better like custom flags)? If not we probably want to at least put it in the docs.

Reply:
We could make it a pytest session fixture, so it would be executed once at each test run, but I am not sure it is good practice though. I think a "contribute" section in the README would be more relevant. What do you think?

Reply:
Yeah, just moving into the docs seems fine to me.